cql3/statements/select_statement.cc

```diff
@@ -1629,7 +1629,10 @@ std::unique_ptr<prepared_statement> select_statement::prepare(data_dictionary::d
     // using `forward_service`.
     auto can_be_forwarded = [&] {
         return selection->is_aggregate()        // Aggregation only
-            && selection->is_count()            // Only count(*) selection is supported.
+            && ( // SUPPORTED PARALLELIZATION
+                selection->is_count() ||
+                selection->is_uda_reducable()
```

Comment:
So these two conditions are really one.

Reply:
Fiexd conditions and separated them by parallelization version (only COUNT(*) / all aggregates)